packages/calypso-build/webpack/transpile.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Return a webpack loader object containing our jsx (jsx -> js) stack.
+ *
+ * @param  {Object}    _                    Options
+ * @param  {integer}   _.workerCount        Number of workers that are being used by the thread-loader
+ * @param  {string}    _.configFile         Babel loader config file
+ * @param  {string}    _.cacheDirectory     Babel loader cache directory
+ * @param  {string}    _.cacheIdentifier    Babel loader cache identifier
+ * @param  {regEx}    _.exclude    		    Directories to exclude when looking for files to transpile
+ *
+ * @return {Object}   webpack loader object
+ */
+module.exports.loader = ( {
+	workerCount,
+	configFile,
+	cacheDirectory,
+	cacheIdentifier,
+	exclude,
+} ) => {
+	if ( ! workerCount ) {
+		workerCount = 1;
+	}
+
+	if ( ! exclude ) {
+		exclude = /node_modules\//;
+	}
```

Comment:
I think you can set these defaults as part of the function definition :slightly_smiling_face:

```js
module.exports.loader = ( {
	workerCount = 1,
/*...*/
} ) => {
```
etc :slightly_smiling_face:

Reply:
I'm pretty hesitant to add default arguments at this point. Default arguments here are tricky, consumers may not pass an argument because they may want some sort of default behavior. I'd argue that we really don't know what sensible defaults are and they'd be best avoided for now.

When I suggested erroring if we don't get our expected inputs I was thinking that might help surface errors early as we work on this. I was specifically thinking of erroring if we didn't receive `configFile`. With our current assumptions, that may still be a good idea, but isn't a blocker.

Reply:
Okay, makes sense. Let's remove the defaults altogether then, for now.

Reply:
:+1: